web/source/keymanweb.js

```diff
@@ -3060,8 +3072,7 @@
   **/
   keymanweb.setDfltKeyboard=function()
   {
-    keymanweb._SetActiveKeyboard('Keyboard_us','eng',true);
-    keymanweb.doKeyboardChange('Keyboard_us','eng',true);
+    keymanweb.setActiveKeyboard(''); // This line is used in all the UI elements to set the default keyboard, so we should follow suit here.
```

Comment:
It's not turning off KeymanWeb entirely - it's merely returning to the default keyboard.  It appears (if I had to guess) that at some point "Keyboard_US" was loaded every time as the default, but it is now accessed as a blank-named default.

If it fails, the user can easily select a different language and keep using KeymanWeb without issue.

This being stated, it does not appear to be in our published API on help.keyman.com, so we could inline it and eliminate the method if desired.